examples/trusted_information_retrieval/example.toml

```diff
@@ -17,6 +17,7 @@ module_1 = { Cargo = { cargo_manifest = "examples/trusted_information_retrieval/
 [server]
 additional_args = [
   "--config-files=config=examples/trusted_information_retrieval/database.toml",
+  "--signature-manifest=examples/trusted_information_retrieval/signature.toml"
```

Comment:
In general, this file name contains signatures (plural), right? So not sure why it's called signature (singular).